src/main/java/com/uid2/attestation/gcp/OidcAttestationProvider.java

```diff
@@ -0,0 +1,32 @@
+package com.uid2.attestation.gcp;
+
+import com.uid2.enclave.AttestationException;
+import com.uid2.enclave.IAttestationProvider;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class OidcAttestationProvider implements IAttestationProvider {
+	// the local file contains OIDC token
+	private String tokenFilePath = "/run/container_launcher/attestation_verifier_claims_token";
+
+	@Override
+	public byte[] getAttestationRequest(byte[] publicKey) throws AttestationException {
```

Comment:
Is this work in progress? I assume publickKey will be used later?

Reply:
It's completed, OIDC doesn't need public key

Reply:
I added a SuppressWarning("unused") annotation to make it clearer